str id
nav_msgs/Odometry pose
geometry_msgs/Twist twist
int[] past_vel
int d
int[] d
bool stop
geometry_msgs/Point[] future_waypoints